ansible/roles/postldap/defaults/main.yml

```diff
@@ -321,10 +320,34 @@ postldap__postfix__dependent_lookup_tables:
                        (authorizedService=mail:receive)
                      )
                    )'
-    result_attribute: 'mailForwardTo'
+    result_attribute: ''
```

Comment:
I'm not sure if an empty `result_attribute` needs to be set? What happens if it's omitted entirely from the lookup table?

Reply:
If I remove the corresponding line in the final lookup file, it is still retrieving working without complaint using `postmap`.
It should be safe to remove all the empty `result_attribute` entries.